makerdao/governance-portal-v2
# Add unique key to duplicate delegate event check

### What does this PR do?

The issue was that we performed a duplicate check across all delegation events and filtered out if the tx already existed in the array

With multi-send delegation, there can be multiple events within the same tx that are valid

But we also have an issue where some events appeared twice in the db

To fix this, we can update the filter to scope the duplicate check to the address, so duplicate hash+address won't show, but multiple addresses in the same tx will still show

To test:

See the address that has duplicate db records for the same address+hash only shows 1 record: http://localhost:3000/address/0xc54c3691419c2996eb6e76b421e7f1132ab0ee7a

But the delegator who used multi-send now shows all the delegation events: http://localhost:3000/address/0xf65475e74c1ed6d004d5240b06e3088724dfda5d
